### ISSUE TITLE
[Error] [TSSC] checking tss status failed!

### ISSUE THREAD

**Issue by ghost** [encounter/tsschecker#5]
C:\Users\MekdiPro\Desktop\tsschecker_win64_v268>tsschecker.exe -d iPhone6,1 -e 12345678 -B n51ap -i 8.0 -s
Version: 9ce5e00fdeaba8d7e763a028d32b11937839d2f9 - 268
[TSSC] manually specified ecid to use, parsed "ecid here" to dec: "dec here" hex: "hex here"   (i left out the ecid)
[TSSC] opening firmware.json
[JSON] counting elements
[JSON] parsing elements
[TSSC] got firmwareurl for iOS 8.0 build 12A365
[TSSC] opening Buildmanifest for iPhone6,1_8.0_12A365
[LFZP] downloading BuildManifest.plist from http://appldnld.apple.com/iOS8/031-05671.20140917.dyJlj/iPhone6,1_8.0_12A3
65_Restore.ipsw

[Error] [LFZP] failed to open url
[Error] [TSSC] ERROR: could not get BuildManifest for firmwareurl http://appldnld.apple.com/iOS8/031-05671.20140917.dy
Jlj/iPhone6,1_8.0_12A365_Restore.ipsw

[Error] [TSSC] checking tss status failed!

C:\Users\MekdiPro\Desktop\tsschecker_win64_v268>

FYI, this works on mac so can you fix this for windows

**Comment by encounter** [comment]
This may be due to Apple's servers being slow. Keep trying.

**Comment by ghost** [comment]
is the server for mac and window different? coz I try to save blob on mac....works after attempt 14 but on windows just fails. I have tried multiple times and every time on mac works but windows fails....tried it as i was typing this comment.

**Comment by encounter** [comment]
No, they're not different. I just tried your exact command on my W10 machine and it works, oddly.

**Comment by ghost** [comment]
I am on window 7, does that affect it?

**Comment by encounter** [comment]
It might. I haven't had the chance to try it on W7 yet.

**Comment by ghost** [comment]
any chance you tried tsschecker on Win7?
